entries/z/zola.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Zola": {
+    "domain": "zola.com",
+    "tfa": [
+      "sms",
+      "email"
+    ],
+    "documentation": "https://www.zola.com/expert-advice/a-security-update-for-our-couples",
+    "categories": [
+      "tickets"
```

```suggestion
      "hosting",
      "tickets"
```

Hosting is our category for online website hosting, which Zola offers. I don't know about tickets and events category for them though. Maybe task management would be better?

---

I went with this because, according to the [category descriptions](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#categories), the `tickets` category represents "Ticketing and event platforms". I would consider a wedding planning site an "event platform". 🤷‍♂️ I'm amenable to adding `task` too, though I would picture that being used more for "general" task management sites or sites like Trello. This one is very wedding-specific.

---

Makes sense. Tickets is good. What is your opinion on the hosting category?

---

I suppose it's fine, since the custom website thing is a big part of their offering. People aren't really "hosting" their own websites here with like custom code/domains; Zola just provides a set of web _pages_ that users can enable/disable with limited text/color customization. But that is technically still a hosted website. 🤷‍♂️